services/121-service/src/payments/fsp-integration/safaricom/safaricom.service.ts

```diff
@@ -30,36 +26,6 @@ export class SafaricomService
     private readonly redisClient: Redis,
   ) {}
 
-  /**
-   * Do not use! This function was previously used to send payments.
-   * It has been deprecated and should not be called anymore.
-   */
-  // ##TODO: remove this method. What is the point of keeping it?
-  public async sendPayment(
```

Comment:
@diderikvw  in an earlier iteration of this PR I actually suggested to remove this, and you then said we should keep it. See https://github.com/global-121/121-platform/pull/6303#discussion_r1893694285.

I am still fine with removing it, so up to you.

Reply:
@jannisvisser Yeah, I remember Peter and I also removed this function in IntersolveVisaService at first, and removed the implements interface directive. But then later put it back. I do not remember why anymore, and if no linting and no automated tests complain, my LOGAF is low on this one. So let's just let it be removed.